app/models/observation.rb

```diff
@@ -345,12 +345,59 @@ def owners_votes
   # Get a list of this User's Votes for this Observation.
   def users_votes(user)
     result = []
-    for n in namings
+    namings.each do |n|
       if v = n.users_vote(user)
         result << v
       end
     end
-    return result
+    result
+  end
+
+  # list of Owners's "favorite" votes for this Observation
+  def owner_favorite_votes
+    votes.where(user_id: user_id, favorite: true)
+  end
+
+  def owner_favorite_vote
+    owner_favorite_votes[0]
```

Comment:
I prefer .first over [0] in a case like this.  [0] gives me the willies since in many languages it's an error if the list is empty.  In Ruby it's OK, so this isn't a blocker.